testnet/stacks-node/src/event_dispatcher.rs

```diff
@@ -1272,6 +1266,12 @@ impl EventDispatcher {
         block_timestamp: Option<u64>,
         coinbase_height: u64,
     ) {
+        let interested_observers =
```

Comment:
Should this get passed into `create_dispatch_matrix_and_event_vector`? and used there? Otherwise, I think this is still sending to observers that should have been filtered out.

Reply:
Actually, I think this new code can just be removed.